llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -340,6 +340,10 @@ static cl::opt<bool>
                            cl::desc("Prefer in-loop vector reductions, "
                                     "overriding the targets preference."));
 
+cl::opt<bool> PreferStridedLoadStore("prefer-strided-load-store",
```

Comment:
Why do we need a flag? Is it not sufficient for targets to indicate that they should use strided loads?